tests/gis_tests/geoapp/test_functions.py

```diff
@@ -104,7 +104,7 @@ def test_asgml(self):
         if oracle:
             # No precision parameter for Oracle :-/
             gml_regex = re.compile(
-                r'^<gml:Point srsName="SDO:4326" xmlns:gml="http://www.opengis.net/gml">'
+                r'^<gml:Point srsName="EPSG:4326" xmlns:gml="http://www.opengis.net/gml">'
```

Comment:
This seems to be a change in Oracle 12. Oracle 11 works with SDO but also doesn't pass the `self.assertIn` later. This affects the deprecated GeoQuerySet test also. I'm happy to merge this as is and leave it up to you as to whether or not it's worth fixing separately.
```
======================================================================
FAIL: test_asgml (gis_tests.geoapp.test_functions.GISFunctionsTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/tim/code/django/django/test/testcases.py", line 1119, in skip_wrapper
    return test_func(*args, **kwargs)
  File "/home/tim/code/django/tests/gis_tests/geoapp/test_functions.py", line 116, in test_asgml
    self.assertTrue(gml_regex.match(ptown.gml))
AssertionError: None is not true

======================================================================
FAIL: test_gml (gis_tests.geoapp.tests.GeoQuerySetTest)
Testing GML output from the database using GeoQuerySet.gml().
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/tim/code/django/django/test/testcases.py", line 1119, in skip_wrapper
    return test_func(*args, **kwargs)
  File "/home/tim/code/django/tests/gis_tests/geoapp/tests.py", line 645, in test_gml
    self.assertTrue(gml_regex.match(ptown.gml))
AssertionError: None is not true
```